File: src/main/java/ar/edu/um/tesoreria/rest/model/ClaseChequera.java
/**
 * 
 */
package ar.edu.um.tesoreria.rest.model;

import java.io.Serializable;

import jakarta.persistence.Column;
import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.EqualsAndHashCode;
import lombok.NoArgsConstructor;

/**
 * @author daniel
 *
 */
@Data
@Entity
@Table
@EqualsAndHashCode(callSuper = false)
@NoArgsConstructor
@AllArgsConstructor
public class ClaseChequera extends Auditable implements Serializable {
	/**
	 * 
	 */
	private static final long serialVersionUID = 7108406387093539108L;

	@Id
	@Column(name = "cch_id")
	private Integer claseChequeraId;

	@Column(name = "cch_nombre")
	private String nombre;
	
}

File: src/main/java/um/tesoreria/rest/kotlin/model/ClaseChequera.kt
package um.tesoreria.rest.kotlin.model

import um.tesoreria.rest.model.Auditable
import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table
data class ClaseChequera(

    @Id
    @Column(name = "cch_id")
    var claseChequeraId: Int? = null,

    @Column(name = "cch_nombre")
    var nombre: String? = null,

    var preuniversitario: Byte = 0,
    var grado: Byte = 0,
    var posgrado: Byte = 0,
    var curso: Byte = 0,
    var secundario: Byte = 0,

    ) : Auditable()
